src/Stratis.Bitcoin/Base/ChainRepository.cs

```diff
@@ -43,22 +43,30 @@ public Task LoadAsync(ConcurrentChain chain)
             {
                 using (DBreeze.Transactions.Transaction transaction = this.dbreeze.GetTransaction())
                 {
+                    transaction.ValuesLazyLoadingIsOn = false;
                     ChainedBlock tip = null;
+                    BlockHeader previousHeader = transaction.Count("Chain") != 0 ?
+                        transaction.Select<int, BlockHeader>("Chain", 0).Value : null;
```

Comment:
i think this is not a correct way of doing that
you should get Row and check if it exists
just because the table is not empty, it does not mean there is value stored with key 0